system/web/context/ExceptionBean.cfc

```diff
@@ -35,7 +35,7 @@ component accessors="true"{
 		any extraMessage = "",
 		any extraInfo = ""
 	){
-		variables.exceptionStruct = arguments.errorStruct;
+		variables.exceptionStruct = duplicate(arguments.errorStruct);
```

Comment:
CF-11 converting it into object 
--isObject(arguments.errorStruct) YES 
--isStruct(arguments.errorStruct) NO
Manually I can add info into struct 
```
if(isObject ) then
variables.exceptionStruct.Message = arguments.errorStruct.getMessage
--- etc
```
its really strange why CF-11 does it :( 

Lucee works fine 

I don't have CF-10 at the moment, just picking up code-base after a long time :)